syntax = "proto3";

//define PIN service
service PIN {
  rpc SendPin (Pin) returns (Acknowledgement) {}
}

//define map request message
message Pin {
  string pin = 1;
}

//response message containing the map features
message Acknowledgement {
  string ack = 1;
}